src/vsr/replica.zig

```diff
@@ -7114,7 +7115,7 @@ pub fn ReplicaType(
                 // "`replica.op` exists" invariant may be broken briefly between
                 // set_op_and_commit_max() and replace_header().
 
-                self.set_op_and_commit_max(header_head.op, commit_max, "on_do_view_change");
+                self.set_op_and_commit_max(header_head.op, commit_max, @src());
```

Comment:
This was `on_do_view_change`, now it is `primary_set_log_from_do_view_change_messages`